### PR TITLE
V1.0.14.0

### PR DESCRIPTION
**Pull request by sra122**
Major changes in the version are
1. Migrating data from Settings Repository to PM Database.
2. Displaying the successfully pushed products to PandaBlack in the Report Section.

### CODE REVIEW

File: meta/documents/changelog_de.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Release Notes für PANDA.BLACK
+
+## v1.0.14
```
Comment (jschaakplenty) [inline meta/documents/changelog_de.md]: This is no longer a small patch, but either a small or major change. The versioning should reflect that.

File: meta/documents/changelog_en.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Release notes for PANDA.BLACK
 
+## v1.0.14
```
Comment (jschaakplenty) [inline meta/documents/changelog_en.md]: This is no longer a small patch, but either a small or major change. The versioning should reflect that.

File: plugin.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name"              : "PandaBlack",
   "namespace"         : "PandaBlack",
   "type"              : "general",
-  "version"           : "1.0.13",
+  "version"           : "1.0.14",
```
Comment (jschaakplenty) [inline plugin.json]: This is no longer a small patch, but either a small or major change. The versioning should reflect that.

File: meta/documents/changelog_de.md (hunk 1 above)
Comment (sra122) [reply meta/documents/changelog_de.md]: Could you elaborate your suggestions please.

Comment (jschaakplenty) [reply meta/documents/changelog_de.md]: See https://en.wikipedia.org/wiki/Software_versioning#Degree_of_compatibility

Comment (sra122) [reply meta/documents/changelog_de.md]: Thank you :). I will make changes accordingly.

Comment (sra122) [reply meta/documents/changelog_de.md]: Hi 
We made a new pull request for new version. It differs only with the version with current pull request. Ref: https://github.com/sra122/PandaBlack/pull/7

File: meta/documents/changelog_en.md (hunk 2 above)
Comment (jschaakplenty) [reply meta/documents/changelog_en.md]: Done

File: plugin.json (hunk 3 above)
Comment (jschaakplenty) [reply plugin.json]: Done

File: meta/documents/changelog_de.md (hunk 1 above)
Comment (jschaakplenty) [reply meta/documents/changelog_de.md]: Done

File: src/Crons/CategoriesAndAttributesUpdateCron.php [hunk 4]
```diff
@@ -2,13 +2,118 @@
...
+use PandaBlack\Helpers\SettingsHelper;
+use PandaBlack\Repositories\AttributesRepository;
+use PandaBlack\Repositories\AttributeValuesRepository;
+use PandaBlack\Repositories\CategoriesRepository;
 use Plenty\Modules\Cron\Contracts\CronHandler as Cron;
 
 class CategoriesAndAttributesUpdateCron extends Cron
 {
-    public function __construct(AttributeController $attributeController)
+    public function __construct()
```
Comment (mbrueschke) [inline src/Crons/CategoriesAndAttributesUpdateCron.php]: The logic of the cron should not run in the __construct(). There is a method called handle() which should be used. 
Like: 

public function handle(){
   $this->updadeCategories();
   $this->updateAttributes();
}

Comment (sra122) [reply src/Crons/CategoriesAndAttributesUpdateCron.php]: So you mean, handle method will be the default function that will be called like Constructor whenever the "CategoriesAndAttributesUpdateCron" class is called. Or do I need to specify the method name in the "Service Provider" ??

Comment (mbrueschke) [reply src/Crons/CategoriesAndAttributesUpdateCron.php]: Exactly, you do not have to do smth special to trigger the handle. It is automatically called

Comment (sra122) [reply src/Crons/CategoriesAndAttributesUpdateCron.php]: Ok Thank you

Comment (sra122) [reply src/Crons/CategoriesAndAttributesUpdateCron.php]: You can find the changes

**Pull request closed without merging**